tests/unit/ValueObjects/DownloadStatisticsTest.php

```diff
@@ -18,8 +18,8 @@ public function itCanCreateADownloadStatistic()
         $downloadStatistic = new DownloadStatistics(
             $package,
             13,
-            new Chronos('1989-12-13'),
-            new Chronos('1988-11-07')
+            new Chronos('1989-12-13T00:00:00+00:00'),
```

Comment:
Fyi, probably wouldn't have done this either but tests were failing on my machine because of my php installation's timezone settings... Alternatively you could set the php timezone setting in `setUp` so it's the same for everyone.